djangoproject/templates/base_community.html

```diff
@@ -6,11 +6,8 @@
 
 {% block header %}
   <h1 class="visuallyhidden">Community</h1>
-  <p><em>Django</em> Community{% if community_stats %}
-  {% if community_stats.people %} <em>{{ community_stats.people }}</em> people,{% endif %}
-  {% if community_stats.countries %} <em>{{ community_stats.countries }}</em> countries,{% endif %}
-  {% if community_stats.packages %} <em>{{ community_stats.packages }}</em> packages and projects.{% endif %}
-  {% endif %}</p>
+  <p>Building the <em>Django</em> Community{% if community_stats %}
+  {% if community_stats.age %} for <em>{{ community_stats.age }}</em>{% endif %}{% endif %}. Come join us!</p>
```

Comment:
I would totally remove stats:
```suggestion
  <p><em>Django</em> Community. Come join us!</p>
```

Reply:
I quite liked the 15 years thing... — It's Authority building, like the current stats. It says _This is a big project_ you can trust it_. (like those client logos people put on their web sites.) 

If you don't buy that we can drop it. 😀

Reply:
`timesince()` is translated so it is not consistent, we should unlocalize it

![obraz](https://user-images.githubusercontent.com/2865885/92468806-08219200-f1d4-11ea-8cf0-1321887a2871.png)

Reply:
OK, I'll write a test for that.

Reply:
OK, updated.